From: Edward Cree <ecree@solarflare.com>
Date: Thu, 1 Dec 2016 17:01:07 +0000
Subject: sfc: don't select SFC_FALCON
Patch-mainline: v4.10-rc1
Git-commit: d7d6cabaa1a5295b5a6836b84cc599488ab5c84d
References: bsc#1017967 FATE#321663

Easy enough for Falcon users to enable it when making oldconfig.

Signed-off-by: Edward Cree <ecree@solarflare.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/sfc/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/sfc/Kconfig
+++ b/drivers/net/ethernet/sfc/Kconfig
@@ -6,7 +6,6 @@ config SFC
 	select I2C
 	select I2C_ALGOBIT
 	select PTP_1588_CLOCK
-	select SFC_FALCON
 	---help---
 	  This driver supports 10/40-gigabit Ethernet cards based on
 	  the Solarflare SFC9000-family and SFC9100-family controllers.
